File: ImmersiveStatusBarDemo/app/src/main/java/com/paulniu/immersivestatusbardemo/MainActivity.java
package com.paulniu.immersivestatusbardemo;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;

public class MainActivity extends AppCompatActivity {

    private Button btn01;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        btn01 = findViewById(R.id.btn01);
        btn01.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                Intent intent = new Intent(MainActivity.this,Layout1Activity.class);
                startActivity(intent);
            }
        });
    }
}

File: CameraDemo/app/src/main/java/com/paulniu/camerademo/MainActivity.kt
package com.paulniu.camerademo

import android.Manifest
import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button
import androidx.core.app.ActivityCompat
import com.paulniu.camerademo.camera1.Camera1SurfaceActivity
import com.paulniu.camerademo.camera2.Camera2SurfaceActivity

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        ActivityCompat.requestPermissions(
            this,
            arrayOf(
                Manifest.permission.CAMERA,
                Manifest.permission.WRITE_EXTERNAL_STORAGE,
                Manifest.permission.READ_EXTERNAL_STORAGE
            ),
            0
        )
        val button1 = findViewById<Button>(R.id.button_camera_1)
        val button2 = findViewById<Button>(R.id.button_camera_2)
        button1.setOnClickListener {
            val intent = Intent(this, Camera1SurfaceActivity::class.java)
            startActivity(intent)
        }
        button2.setOnClickListener {
            val intent = Intent(this, Camera2SurfaceActivity::class.java)
            startActivity(intent)
        }
    }

    override fun onRequestPermissionsResult(
        requestCode: Int,
        permissions: Array<out String>,
        grantResults: IntArray
    ) {
        super.onRequestPermissionsResult(requestCode, permissions, grantResults)
    }

}